ingest-app/src/migrations/006_update_bulk_update_tables.clj

```diff
@@ -0,0 +1,87 @@
+(ns migrations.006-update-bulk-update-tables
+  (:require [clojure.java.jdbc :as j]
+            [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 6."
+  []
+  (println "migrations.006-update-bulk-update-tables up...")
+  (j/with-db-transaction [trans-conn (config/db)]
+    (j/db-do-commands trans-conn "LOCK TABLE CMR_INGEST.bulk_update_task_status IN EXCLUSIVE MODE")
+    (j/db-do-commands trans-conn "LOCK TABLE CMR_INGEST.bulk_update_coll_status IN EXCLUSIVE MODE")
+    (j/db-do-commands trans-conn "CREATE TABLE CMR_INGEST.bulk_update_task_status_new (
+                                  TASK_ID VARCHAR(255) NOT NULL,
+                                  PROVIDER_ID  VARCHAR(10) NOT NULL,
+                                  REQUEST_JSON_BODY  BLOB NOT NULL,
+                                  STATUS VARCHAR(20),
+                                  STATUS_MESSAGE   VARCHAR(255),
+                                  CREATED_AT TIMESTAMP(9) WITH TIME ZONE DEFAULT sysdate NOT NULL)")
+    (j/db-do-commands trans-conn "CREATE TABLE CMR_INGEST.bulk_update_coll_status_new (
+                                  TASK_ID VARCHAR(255) NOT NULL,
+                                  CONCEPT_ID VARCHAR(255) NOT NULL,
+                                  STATUS VARCHAR(20),
+                                  STATUS_MESSAGE  VARCHAR(500),
+                                  ACTION_TAKEN VARCHAR2(1) DEFAULT 'F' NOT NULL)")
```

Comment:
What are the possible values for ACTION_TAKEN?

If we're really going to use just a single character CHAR would be better than VARCHAR. In general though I am not a fan of using a single letter to represent things. There's really not much overhead in using a short VARCHAR string for the the action taken especially without a lot of rows. A more descriptive term will just make things easier for future maintainers.